WiserTaskScheduler/WiserTaskScheduler/Core/Services/LogService.cs

```diff
@@ -108,8 +113,15 @@ await databaseConnection.ExecuteAsync(@$"INSERT INTO {WiserTableNames.WtsLogs} (
                             // If writing to the database fails log its error.
                             logger.Log(logLevel, $"Failed to write log to database due to exception: ${e}");
                         }
-
                         logger.Log(logLevel, message);
+
+                        // If Slack channel set use Slack .
+                        if (logLevel == logSettings.SlacklogLevel)
```

Comment:
Deze `if` graag uitbreiden met controle of er wel een Slack Access Token is ingesteld. Je hebt namelijk in `ConfigureWtsServices` van `Program.cs` gemaakt dat de service niet wordt geregistreerd wanneer er geen access token is ingevuld. Dus als je nu geen access token invult, dan gaat die alsnog proberen het bericht naar Slack te sturen, maar dan zul je een foutmelding krijgen op `GetRequiredService`, omdat die service niet geregistreerd is.